io.catenax.batch/RELEASE_NOTES.md

```diff
@@ -1,7 +1,17 @@
 # Changelog
 All notable changes to this model will be documented in this file.
 
-## [Unreleased]
+## [2.0.1] 2023-01-04
```

Comment:
please change the date to 4th december if you want MS2 on next weekly.

Reply:
Thanks for the hint! :)